roles/rsyslog/README.md

```diff
@@ -21,53 +46,96 @@ localhost ansible_user=YOUR_ANSIBLE_USER openshift_logging_use_ops=True
 localhost ansible_user=YOUR_ANSIBLE_USER openshift_logging_use_ops=True
 ```
 
-2. vars.yaml stores variables which are passed to ansible to control the tasks.
+vars.yaml
+---------
+1. vars.yaml stores variables which are passed to ansible to control the tasks.
 
-   Currently, this rsyslog roles support 2 deployment variables, rsyslog__viaq and rsyslog__example.  I.e., there are 3 sets of deployment - rsyslog__viaq: true, rsyslog__example: true, and both false.  Theoretically, both true could be set and the specified configuration files are deployed, but rsyslog does not work properly with the configuration.
+   Currently, this rsyslog roles support 2 deployment variables, rsyslog_viaq and rsyslog_example.  I.e., there are 3 sets of deployment - rsyslog_viaq: true, rsyslog_example: true, and both false.  Theoretically, both true could be set and the specified configuration files are deployed, but rsyslog does not work properly with the configuration.
 
    See the variables section for each variable.
 
    Sample vars.yaml file for the viaq case.
 ```
-rsyslog__enabled: true
+rsyslog_enabled: true
 # install viaq packages & config files
-rsyslog__viaq: true
-rsyslog__capabilities: [ 'viaq' ]
-rsyslog__group: root
-rsyslog__user: root
+rsyslog_viaq: true
+rsyslog_capabilities: [ 'viaq' ]
+rsyslog_group: root
+rsyslog_user: root
 elasticsearch_server_host: es_hostname
 elasticsearch_server_port: 9200
 ```
 
-2.1  vars.yaml to configure to handle the inputs from openshift containers
+2. vars.yaml to configure to handle the inputs from openshift containers
 ```
-rsyslog__enabled: true
+rsyslog_enabled: true
 # install viaq packages & config files
-rsyslog__viaq: true
-rsyslog__capabilities: [ 'viaq', 'viaq-k8s' ]
-rsyslog__group: root
-rsyslog__user: root
-# If 'viaq-k8s' is in rsyslog__capabilities, logging_mmk8s_* need to be specified.
-logging_mmk8s_token: "{{rsyslog__viaq_config_dir}}/mmk8s.token"
-logging_mmk8s_ca_cert: "{{rsyslog__viaq_config_dir}}/mmk8s.ca.crt"
-# If use_omelasticsearch_cert is True, logging_elasticsearch_* need to be specified.
+rsyslog_viaq: true
+rsyslog_capabilities: [ 'viaq', 'viaq-k8s' ]
+rsyslog_group: root
```

Comment:
Why not set rsyslog_unprivileged=false instead of setting the group and user?
@nhosoi @richm